src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/EnvelopeProcessor.java

```diff
@@ -147,6 +151,32 @@ public static void assertEnvelopeHasPdfs(InputEnvelope envelope, List<Pdf> pdfs)
         }
     }
 
+    /**
+     * Assert scannable item contains ocr data
+     * when envelope classification is NEW_APPLICATION
+     * and jurisdiction is SSCS
+     * and document type of the scannable item is SSCS1.
+     * Throws exception otherwise.
+     *
+     * @param envelope to assert against
+     */
+    public void assertEnvelopeContainsOcrDataIfRequired(InputEnvelope envelope) {
+
+        if (envelope.jurisdiction.equalsIgnoreCase("SSCS")
+            && Classification.NEW_APPLICATION.equals(envelope.classification)) {
+
+            boolean ocrDataExists = envelope.scannableItems
+                .stream()
+                .filter(item -> item.documentType.equals(InputDocumentType.SSCS1))
+                .noneMatch(item -> MapUtils.isEmpty(item.ocrData));
```

Comment:
that doesn't mean ocr data exists, when there are no SSCS1 documents (ocr can be in a document of different type)